### PR TITLE
fix: redraw issue with virtual scroll while dragging scrollbar 

### PR DESCRIPTION
**Pull request by chintankavathia**
**What kind of change does this PR introduce?** (check one with "x")

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
https://code.siemens.com/simpl/simpl-element/-/issues/401

**What is the new behavior?**
User will not have to explicitly move scrollbar to see blank rows updated after doing scroll 

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by chintankavathia** [comment]
Hi @timowolf , @fh1ch  Kindly take a look at this PR
Thank you

File: projects/ngx-datatable/src/lib/components/body/body.component.ts [hunk 1]
```diff
@@ -415,7 +415,18 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
 
     if (direction !== undefined && !isNaN(offset) && offset !== this._offsetEvent) {
       this._offsetEvent = offset;
-      this.page.emit({ offset });
+      // if scroll was done by mouse drag make sure previous row and next row data is also fetched if its not fetched
+      if(scrollInBetween && this.scrollbarV && this.virtualization && this.externalPaging) {    
+        const upRow = this.rows[this.indexes.first - 1];
+        if(!upRow && direction === 'up')
```
Comment (timowolf) [inline projects/ngx-datatable/src/lib/components/body/body.component.ts:421]: do not use if without brackets. run a ng lint --fix to align to eslint rules

**Review by timowolf — CHANGES_REQUESTED** [review]
Many thanks, looks good to me in general. Just some ESLint coding style is not fulfilled.

File: projects/ngx-datatable/src/lib/components/body/body.component.ts [hunk 2]
```diff
@@ -406,7 +406,7 @@ export class DataTableBodyComponent implements OnInit, OnDestroy {
    */
   updatePage(direction: string): void {
     let offset = this.indexes.first / this.pageSize;
-
+    const scrollInBetween = !Number.isInteger(offset);
```
[the remark this answers is not in the document]
Comment (chintankavathia) [reply projects/ngx-datatable/src/lib/components/body/body.component.ts]: Fixed

File: projects/ngx-datatable/src/lib/components/body/body.component.ts (hunk 1 above)
Comment (chintankavathia) [reply projects/ngx-datatable/src/lib/components/body/body.component.ts]: Fixed

**Review by timowolf — APPROVED** [review]
Looks very good to me. Many thanks

**Comment by timowolf** [comment]
@chintankavathia Many thanks for the great contribution

**Pull request merged**